db/seeds.rb

```diff
@@ -6,12 +6,15 @@ def self.start
   def generate
     create_categories
     create_stickers
+    create_users
+    create_orders
   end
 
   def create_categories
     @javascript_category = Category.create(title: "javascript")
     @ruby_category = Category.create(title: "ruby")
     @browser_category = Category.create(title: "browser")
+    puts "#{Category.all.map(&:title).join(", ")} categories created."
```

Comment:
Prefer single-quoted strings inside interpolations.<br>Do not write to stdout. Use Rails' logger if you want to log.